src/picongpu/include/particles/particleToGrid/derivedAttributes/Counter.hpp

```diff
@@ -52,6 +52,6 @@ namespace derivedAttributes
         /* return attribute */
         return particleCounter;
     }
-} /* namespace derivedAttributes */
-} /* namespace particleToGrid */
-} /* namespace picongpu */
+}
```

Comment:
Sorry :pray: for having sparked this discussion again  @ax3l , @psychocoderHPC   
I understand both points - more than once I found having these end-of-namespace comments convenient for reading - but it's more work and can bear the danger of being outdated